pyannote/core/utils/types.py

```diff
@@ -0,0 +1,15 @@
+from typing import Hashable, Union, Tuple, Iterator
+
+from typing_extensions import Literal
```

Comment:
`Literal` is part of Python 3.8 `typing`.
Shouldn't we try `from typing import Literal` and fallback to `typing_extensions` if it fails?

Reply:
https://github.com/python/typing/blob/8280de241fd8c8afe727c7860254b753e383b360/typing_extensions/src_py3/typing_extensions.py#L540 the lib is smart enough to use the default "standard lib" typings, so this logic might not be needed. Your call.